versions/0.0.0-licenses.txt

```diff
@@ -0,0 +1,24 @@
+ http://creativecommons.org/publicdomain/zero/1.0
+ http://creativecommons.org/licenses/by-nc/3.0
+ http://creativecommons.org/licenses/by-nc-sa/3.0
+ http://creativecommons.org/licenses/by/4.0
+ http://creativecommons.org/licenses/by-sa/4.0/
+ http://creativecommons.org/licenses/by-nc/4.0/
+ http://creativecommons.org/licenses/by-nd/4.0/
+ http://creativecommons.org/licenses/by-nc-nd/4.0/
+ http://creativecommons.org/licenses/by-nc-sa/4.0/
+ http://opensource.org/licenses/BSD-2-Clause
+ http://opensource.org/licenses/BSD-3-Clause
+ http://opensource.org/licenses/MIT
+ http://www.apache.org/licenses/LICENSE-2.0
+ http://www.cecill.info/licences/Licence_CeCILL_V2-en.html
+ http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+ http://www.gnu.org/licenses/gpl-3.0.en.html
+ http://www.gnu.org/licenses/lgpl-3.0.txt
+ http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html
+ http://www.mozilla.org/en-US/MPL/2.0/FAQ/
+ http://www.ohwr.org/attachments/735/CERNOHLv1_1.txt
+ http://www.ohwr.org/attachments/2388/cern_ohl_v_1_2.txt
+ http://www.ohwr.org/projects/cernohl/wiki
+ http://www.tapr.org/ohl.html (alias for http://www.tapr.org/TAPR_Open_Hardware_License_v1.0.txt, *deprecated*)
+ http://www.tapr.org/TAPR_Open_Hardware_License_v1.0.txt
```

Comment:
addition:
https://opendatacommons.org/licenses/by/1-0/index.html

what is the procedure for adding licenses here?